test/testTargetsApp/Manifest.json

```diff
@@ -18,8 +18,8 @@
   },
   "externalResources": {
     "script": [ 
-        "https://localhost:2000/IDSCloudApp/IDSCloudApp:ext-js/socketcluster/socketcluster-client.min.js",
-        "http://localhost/compiled/source/cv/cv:libs/EventRecorder.js" 
+      "https://localhost:2000/IDSCloudApp/IDSCloudApp:ext-js/socketcluster/socketcluster-client.min.js",
```

Comment:
So here it is a directory with a colon in it, for test purposes?

Reply:
yes because colons are allowed.  Library prefixes are no longer used

Reply:
Ok, thank you!